### DISCUSSION TITLE
Problem with update from 1.1.2 to 1.2.0

### DISCUSSION THREAD

**Discussion by Zordandi** [thomluther/ha-anker-solix#46] [General]
When I try to update from 1.1.2 to 1.2.0 in HACS I get this error messages after restart.
From my understanding, anker_solix needs aiohttp>=3.9.3 and not 3.9.4.
I'm using HomeAssistant 2024.3 with HACS 1.34 in a docker env.

2024-05-01 11:54:36.350 ERROR (SyncWorker_4) [homeassistant.util.package] Unable to install package aiohttp>=3.9.4: ERROR: Cannot install aiohttp>=3.9.4 because these package versions have conflicting dependencies.
ERROR: ResolutionImpossible: for help visit https://pip.pypa.io/en/latest/topics/dependency-resolution/#dealing-with-dependency-conflicts
2024-05-01 11:54:36.352 ERROR (MainThread) [homeassistant.setup] Setup failed for custom integration 'anker_solix': Requirements for anker_solix not found: ['aiohttp>=3.9.4'].

**Comment by thomluther** [comment]
Hi, I changed the aiohttp requirements back to 3.9.3 in version 1.2.0. In git the 1.2.0 tag is showing for custom_components/anker_solix/manifest.json:
```
  "requirements": [
    "aiohttp>=3.9.3",
    "cryptography>=3.4.8"
  ],
  "version": "1.2.0"
```
This was done base on issue #42 
So I'm not sure where you still got 3.9.4 requirement from. Maybe you have to enforce HACS updating the anker-solix repository information to get proper hash for updated 1.2.0 tag. (HACS refreshes repo info only ever 48h automatically).

**Comment by Zordandi** [comment]
I finally found a way to solve this issue: After the update process in HACS showed the needed restart I took a look at manifest.json and it reads "aiohttp>=3.9.4". So I manually changed it to 3.9.3, restarted HACS and everything works fine.
I know very little about GitHub, but when I compare versions 1.1.2 and 1.2.0 at "https://github.com/thomluther/hacs-anker-solix/compare/1.1.2...1.2.0", it also says 3.9.4 and not your last change to 3.9.3! Maybe that's why it does not work?

**Comment by thomluther** [comment]
This was fixed with new release tag 1.2.1.
